web/tests/people.test.js

```diff
@@ -662,6 +695,19 @@ test_people("filtered_users", () => {
     assert.ok(filtered_people.has(linus.user_id));
 });
 
+test_people("filtered_humans", () => {
+    people.add_active_user(bot_botson);
+    people.add_active_user(bob);
+
+    const humans = people.get_realm_active_human_users();
+    const search_term = "b";
+    const filtered_people = people.filter_people_by_search_terms(humans, [search_term]);
+
+    assert.equal(filtered_people.size, 1);
```

Comment:
I am not sure whether this test is necessary, Should be fine to just have a test that `get_realm_active_human_users` does not return bots.